nanostack-border-router.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/nanostack-border-router/#9a0b032e025fb06675eb165df2a45a4ef5e15d64
```

Comment:
I don't think this is right.
You need to update this once the https://github.com/ARMmbed/nanostack-border-router/pull/20/ is in.